citi-onboarding/hacka-pride
# Feature/front contact section

## What I did
* Added the contact section
* Added the message box with the necessary requirements to nodemailer
* Added footer

## How to test
1.  Fill `.evn` template posted on notion
2. Run the command `yarn start` on `/client`. At the same time, open another cmd and run the comand `nodemon` on `/server`.
3. If it works, the contact form information input must be seen in your `EMAIL_CLIENT` you field at `.env`.
4. Verify the responsivity
